tests/unit/worker/test_submission_worker.py

```diff
@@ -27,27 +31,50 @@
 
 class BaseAPITestClass(APITestCase):
     def setUp(self):
+
         self.BASE_TEMP_DIR = tempfile.mkdtemp()
+
+        self.CHALLENGE_DATA_DIR = join(
+            self.BASE_TEMP_DIR,
+            "compute/challenge_data/challenge_{challenge_id}",
+        )
+        self.PHASE_DATA_BASE_DIR = join(
+            self.CHALLENGE_DATA_DIR, "phase_data"
+        )
+        self.PHASE_DATA_DIR = join(
+            self.PHASE_DATA_BASE_DIR, "phase_{phase_id}"
+        )
+        self.CHALLENGE_IMPORT_STRING = "challenge_data.challenge_{challenge_id}"
+
         self.temp_directory = join(self.BASE_TEMP_DIR, "temp_dir")
+
+        self.testserver = "http://testserver"
         self.url = "/test/url"
-        self.input_file = open(join(self.BASE_TEMP_DIR, 'dummy_input.txt'), "w+")
+
+        self.input_file = open(
```

Comment:
@nikochiko can we revert this to original code, making it multiline makes it hard to read.

Reply:
Yep. :+1: Added the change